packages/editor-ui/src/mixins/workflowRun.ts

```diff
@@ -185,9 +185,7 @@ export const workflowRun = defineComponent({
 					startNodes.push(nodeName);
 				}
 
-				const isNewWorkflow = this.workflowsStore.isNewWorkflow;
```

Comment:
So, this will not force save all new workflows before executing, but not save workflows that already have an ID, right?

Reply:
This will force-save workflows that have no ID before execution, or more strictly workflows where `id` is the placeholder `'__EMPTY__'`.